README.md

```diff
@@ -2091,6 +2091,27 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
     </details>
 
+* <a id='remove-blank-lines-between-chained-functions'></a>(<a href='#remove-blank-lines-between-chained-functions'>link</a>) **[Long](https://github.com/airbnb/swift#column-width) Remove blank lines between chained functions.**  [![SwiftFormat: blanklinesbetweenchainedfunctions](https://img.shields.io/badge/SwiftFormat-wrapArguments-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#blanklinesbetweenchainedfunctions)
+
+  <details>
+
+  ```swift
+  // WRONG
+  [0, 1, 2]
+    .map { $0 * 2 }
+
+
+
+    .map { $0 * 3 }
```

Comment:
We already have the [consecutiveBlankLines](https://github.com/nicklockwood/SwiftFormat/blob/develop/Rules.md#consecutiveBlankLines) enabled -- let's limit the example to just a single blank line so it doesn't also conflict with that other rule.

```suggestion
    .map { $0 * 2 }

    .map { $0 * 3 }
```